fbfmaproom/fbfmaproom.py

```diff
@@ -71,82 +64,107 @@ def is_valid_root(path):
     return False
 
 
-APP = FbfDash(
-    __name__,
-    external_stylesheets=[dbc.themes.BOOTSTRAP],
-    server=SERVER,
-    url_base_pathname=f"{PFX}/",
-    meta_tags=[
-        {"name": "description", "content": "content description 1234"},
-        {"name": "viewport", "content": "width=device-width, initial-scale=1.0"},
-    ],
-)
-APP.title = "FBF--Maproom"
```

Comment:
I had to move stuff around in the file so that functions that get called at import time are defined before they get called. This is annoying; I would like to switch to using the [application factory](https://flask.palletsprojects.com/en/2.3.x/patterns/appfactories/) approach, and was tempted to do it now, but decided to wait.